benchmarks/nel/scripts/datasets/evaluation.py

```diff
@@ -144,8 +149,8 @@ def report(evaluation_results: Tuple["EvaluationResults"]) -> None:
             eval_result._extend_report_overview_table(overview_table)
             eval_result._extend_report_labels_table(label_table, tuple(labels))
 
-        logger.info(overview_table)
-        logger.info(label_table)
+        logger.info("\n" + str(overview_table))
+        logger.info("\n" + str(label_table))
```

Comment:
This is from an older PR, but I would rename "TPOS", "FPOS" and "FNEG" to "TP", "FP" and "FN" as those terms are more standard.

Also, for the second table, is the main use-case that we want to compare performance on labels within a model, or do we want to compare the performance of one label across models? The table supports the former, but I was kind of expecting the latter, in which case I would sort the entries differently: first for each label show all models and their scores, only then continue to the next label. But it depends on what the main usage of this table is.

Reply:
Haven't really thought about that - I agree that the latter use case makes more sense. Adapted it accordingly & renamed TPOS etc.